ankdown/ankdown.py

```diff
@@ -33,7 +33,7 @@
 ```
 
 Usage:
-    ankdown.py [-r DIR] [-p PACKAGENAME] [--css CSS_STRING] [--dollar]
```

Comment:
I think I favor getting rid of the `--css` and `--dollar` options here; they're both fairly new anyway, and it seems fairly easy to port anything relying on them to use `--config`.

Reply:
👍 I had forgotten to remove `--css`. I did not remove the `--dollar` because it was not "my" argument. I would be for it, too.